src/components/LoginBtn.tsx

```diff
@@ -0,0 +1,24 @@
+import { useSession, signIn, signOut } from 'next-auth/react';
+import { useTranslation } from 'next-i18next';
+
+export default function LoginBtn() {
+  //Will be used in auth
```

Comment:
I didn't get this comment, can you explain it to me?

Reply:
My mistake I should have removed it.